### PR TITLE
✨Switch position measurements

### PR DESCRIPTION
**Pull request by StigLotuft**
Adds the following:
 - Discrete used to model switch positions
 - Example discrete values in both xml and json
 - ValueAliasSet with 4 accompanying values for distinct switch positions
 - MeasurementValueSource for DMS system

Things to consider when reviewing:
 - What terminal should be referenced in Discrete? -> Lowest sequence number
 - Create asset for the sole purpose of connecting it to the measurement? -> not at this stage
 - MeasurementValueQuality is missing -> Covered in a later version
 - Should DMS have a separate Discrete?

Related to issue #159

### CODE REVIEW

File: DIGIN10/Grid/CIMXML/DIGIN10-30-MV1_OP.xml [hunk 1]
```diff
@@ -185,6 +185,100 @@
...
+        <cim:IdentifiedObject.mRID>d007554d-efe2-4f2d-95dd-1439e7a7a228</cim:IdentifiedObject.mRID>
+        <cim:IdentifiedObject.description>Breaker Switch Positions</cim:IdentifiedObject.description>
+        <cim:IdentifiedObject.name>BREAKER SWITCH POSITIONS</cim:IdentifiedObject.name>
+    </cim:ValueAliasSet>
+
+    <cim:ValueToAlias rdf:ID="_521dc3af-f840-4e7f-8543-6811cb2e452a">
+        <cim:IdentifiedObject.mRID>521dc3af-f840-4e7f-8543-6811cb2e452a</cim:IdentifiedObject.mRID>
+        <cim:IdentifiedObject.description>Breaker switch position open</cim:IdentifiedObject.description>
+        <cim:IdentifiedObject.name>OPEN</cim:IdentifiedObject.name>
+        <cim:ValueToAlias.value>0</cim:ValueToAlias.value>
```
Comment (geiabu) [inline DIGIN10/Grid/CIMXML/DIGIN10-30-MV1_OP.xml:214]: My SCADA system uses the following values for position: 
0 - Intermediate
1 - Closed
2 - Open
3 - Fault

**Review by geiabu — COMMENTED** [review]
My SCADA system uses the following values for position:
0 - Intermediate
1 - Closed
2 - Open
3 - Fault

**Comment by StigLotuft** [comment]
> My SCADA system uses the following values for position: 0 - Intermediate 1 - Closed 2 - Open 3 - Fault
> 
> Manual update of switch positions from DMS system (not remote controlled breakers/switches through RTU/SCADA) should have a different property than a breaker that is remotely monitored/controlled. A remote controlled breaker could also be manually updated when connection with RTU or ICCP is faulty

- The values you shared are all reflected in the pr (although not assigned to the same enum value). This could be updated if needed.
- There are two cases when considering DMS. Firstly DMS could also be the source of breaker positions, and secondly it could provide additional information on manually operated breakers. Currently only the first option is modeled. For the second one I am interested in opinions on which device to connect it to? RemoteUnit?
- Interestingly the breaker export of netbas gives these four values -> out, in, intermediate out, intermediate in

**Comment by StigLotuft** [comment]
I have been thinking about this for a few minutes and came to the conclusion that this PR probably does not properly model the scada setup. Typically this case would be rooted in a RTU, however I am not certain that this is useful. The dms case with non-remotely controlled breakers should be modeled correctly at the moment.

**Comment by StigLotuft** [comment]
ValueAliasSet is delayed until later date as agrred in meeting. See #252

**Review by geiabu — APPROVED** [review]

**Review by ThomasRanvikEriksen — APPROVED** [review]

**Pull request merged**